### PR TITLE
clarifying actors and actions...

### PR DESCRIPTION
**Pull request by JustinCappos**
A few changes that I was surprised were left out of the text.  Discovered this was missing when working on #999.  

Also fixed a small typo in the goals portion.

Signed-off-by: Justin Cappos <justincappos@gmail.com>

### CODE REVIEW

File: assessments/guide/self-assessment.md [hunk 1]
```diff
@@ -54,8 +56,30 @@ multiple projects.
...
 domain or problem area.
 
-### Goal
-The intended goal of the projects including the security guarantees the  project
+### Actors
+These are the individual parts of your system that interact to provide the 
+desired functionality.  Actors only need to be separate, if they are isolated
+in some way.  For example, if a service has a database and a front-end API, but
+if a vulnerability in either one would compromise the other, then the distinction
+between the database and front-end are not relevant.
```
Comment (lumjjb) [inline assessments/guide/self-assessment.md:64]:
```suggestion
between the database and front-end is not relevant.
```

File: assessments/guide/self-assessment.md [hunk 2]
```diff
@@ -54,8 +56,30 @@ multiple projects.
...
+
+The means by which actors are isolated should also be described, as this is often
+what prevents an attacker from moving laterally after a compromise.
```
Comment (lumjjb) [inline assessments/guide/self-assessment.md:67]: like actions, if you can add an example definition to illustrate, that would be useful. We should provide a table for them to fill for actors/actions to help provide some guidelines.. like `actor | description | isolation`, and for actions maybe a ` action | actor(s) | description | security considerations`

**Review by lumjjb — COMMENTED** [review]
Thanks for the addition - i made some suggestions of how we can help the consumer fill the self-assessment up

File: assessments/guide/self-assessment.md (hunk 2 above)
Comment (JustinCappos) [reply assessments/guide/self-assessment.md:67]: The rest of this document is pretty sparse.  (I'm actually adding a lot of detail, in comparison.)  Do you think it belongs here or in an example assessment for them to look at?

**Comment by JustinCappos** [comment]
FYI: This is failing linting, etc. due almost entirely to pre-existing text.  This is due to line length, the use of the project "flibble", which isn't a word, etc.  After we merge this, I'll do a pr that fixes those issues.

File: assessments/guide/self-assessment.md (hunk 2 above)
Comment (lumjjb) [reply assessments/guide/self-assessment.md:67]: yea agreed - looks like that’s a gap! Can you help open a ticket to add the the self-assessment template? lgtm this.

**Review by lumjjb — APPROVED** [review]
lgtm . @IAXES can you review this too?

**Review by IAXES — APPROVED** [review]
Looks good: no concerns from me.

**Comment by JustinCappos** [comment]
Okay, so who merges?

**Pull request merged**